src/observable/FromObservable.ts

```diff
@@ -20,7 +20,7 @@ export class FromObservable<T> extends Observable<T> {
     super(null);
   }
 
-  static create<T>(ish: any, mapFnOrScheduler: Scheduler | ((x: any, y: number) => T), thisArg?: any, lastScheduler?: Scheduler): Observable<T> {
+  static create<T>(ish: any, mapFnOrScheduler?: Scheduler | ((x: any, y: number) => T), thisArg?: any, lastScheduler?: Scheduler): Observable<T> {
```

Comment:
this makes sense for signature of generic `from` behavior. however in this case, does array-like observable need to check existence of `mapFn` in line 45? so far I remember arraylikeobservable's mapfn is required parameter. @justinwoo , is my understanding correct? or can we skip `mapFn`?